### PR TITLE
Fix bugs in helix populate tool.

### PR DESCRIPTION
**Pull request by ankagrawal**

### CODE REVIEW

**Comment by jsjtzyy** [comment]
Is the bug about `DelayedRebalancer`?  I see you changed it to `DelayedAutoRebalancer`.

**Comment by ankagrawal** [comment]
> Is the bug about `DelayedRebalancer`? I see you changed it to `DelayedAutoRebalancer`.

Yes there is one about using the correct string for DelayedAutoRebalancer. There is another one in HelixVcrPopulateTool::72. `.withRequiredArg().ofType(Boolean.class);` should be used with the `enableOpt`.

File: ambry-cloud/src/main/java/com/github/ambry/cloud/HelixVcrCluster.java [hunk 1]
```diff
@@ -171,22 +168,4 @@ public void close() {
...
     helixAdmin.close();
   }
-
-  /**
-   * Get the name of the state model for the factory class.
-   * @param stateModelFactoryClass name of the {@link org.apache.helix.participant.statemachine.StateModelFactory} class.
-   * @return state model name corresponding to the factor class.
-   */
-  private String getStateModelNameFromFactory(String stateModelFactoryClass) {
-    if (stateModelFactoryClass.equals(OnlineOfflineHelixVcrStateModelFactory.class.getName())) {
```
Comment (lightningrob) [inline ambry-cloud/src/main/java/com/github/ambry/cloud/HelixVcrCluster.java:181]: Minor: could use equalsIgnoreCase (optional)

Comment (ankagrawal) [reply ambry-cloud/src/main/java/com/github/ambry/cloud/HelixVcrCluster.java:181]: This code has been removed. Each `VcrStateModelFactory` class now maintains its own statemodel name, so there is no longer any need for this comparison anymore.

**Pull request merged**